src/Uno.UI.RemoteControl.VS/EntryPoint.cs

```diff
@@ -406,16 +406,7 @@ private void SetGlobalProperty(Microsoft.Build.Evaluation.Project msbuildProject
 
 		private bool IsApplication(Microsoft.Build.Evaluation.Project project)
 		{
-			var isAndroidApp = project.GetPropertyValue("AndroidApplication")?.Equals("true", StringComparison.OrdinalIgnoreCase) ?? false;
-			var isiOSApp = project.GetPropertyValue("ProjectTypeGuids")?.Equals("{FEACFBD2-3405-455C-9665-78FE426C6842};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}", StringComparison.OrdinalIgnoreCase) ?? false;
-			var ismacOSApp = project.GetPropertyValue("ProjectTypeGuids")?.Equals("{A3F8F2AB-B479-4A4A-A458-A89E7DC349F1};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}", StringComparison.OrdinalIgnoreCase) ?? false;
-			var isExe = project.GetPropertyValue("OutputType")?.Equals("Exe", StringComparison.OrdinalIgnoreCase) ?? false;
-			var isWasm = project.GetPropertyValue("WasmHead")?.Equals("true", StringComparison.OrdinalIgnoreCase) ?? false;
-
-			return isAndroidApp
-				|| (isiOSApp && isExe)
-				|| (ismacOSApp && isExe)
-				|| isWasm;
+			return project.GetPropertyValue("OutputType")?.Equals("Exe", StringComparison.OrdinalIgnoreCase) == true;
```

Comment:
We may want to also add `WinExe`:
https://github.com/unoplatform/uno.templates/blob/f09e1a25e3726e713422dc9f140f82d6fa1635ca/src/Uno.Templates/content/unoapp/MyExtensionsApp._1.Skia.WPF/MyExtensionsApp._1.Skia.WPF.csproj#L3